### PR TITLE
Add createProductKey service function with test

### PR DESCRIPTION
**Pull request by synzen**
Also added support for running database integration tests in the CI environment - so bootiful

### CODE REVIEW

File: src/services/productKeys/createProductKey.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import generateKey from '../../util/generateKey';
+import ProductKey from '../../models/ProductKey';
+
```
Comment (phaseharry) [inline src/services/productKeys/createProductKey.ts]: Why are we manually creating ids? I believe Sequelize automatically assign new instances ids

Comment (synzen) [reply src/services/productKeys/createProductKey.ts]: ah didn't realize that, thanks - removed uuid package, postgres will generate it now

File: package.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
     "@types/jest": "^25.1.4",
     "@types/node": "^13.9.1",
     "@types/supertest": "^2.0.8",
+    "@types/uuid": "^7.0.2",
```
Comment (phaseharry) [inline package.json]: Do we still need the TS types for uuid?

Comment (synzen) [reply package.json]: woopsehs, removed now thanks

**Pull request merged**